client/landing/jetpack-cloud/layout.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import JetpackCloudMasterbar from './masterbar';
+
+class JetpackCloudLayout extends Component {
+	static propTypes = {
+		primary: PropTypes.element,
+		secondary: PropTypes.element,
+	};
+
+	render() {
+		return (
+			<div className="layout is-section-jetpack-cloud">
+				<JetpackCloudMasterbar />
+
+				<div id="content" className="layout__content">
+					<div id="secondary" className="layout__secondary" role="navigation">
+						{ this.props.secondary }
+					</div>
+					<div id="primary" className="layout__primary">
+						{ this.props.primary }
+					</div>
+				</div>
+			</div>
```

Comment:
Should we extract some kind of scaffolding component which takes "primary, secondary, header" as props and includes basic styles for this kinda thing?

Could be then re-used across WordPress.com and Jetpack cloud.

Reply:
Good call! I guess we're still building the basic blocks of it, but once we get to that, it would be a great idea to abstract it 👍